YHP_website/src/main/webapp/WEB-INF/pages/account.jsp

```diff
@@ -57,43 +57,68 @@
 						{
 							alert(response.error);
 						}
-						else
-						{
-							alert("success");
-						}
 					}
 				}
 				xhttp.send('user=' + JSON.stringify(formData));
 			}
+		
 		</script>
 	</head>
 	<body>
 		<div class="container-fluid">
 			<%@ include file="./includes/header.jspf" %>
+			<%!
+			private String getUserAttribute(boolean isEditing, String attributeName, String attributeValue)
+			{
+				final String formType = isEditing? "":" text";
+				final String readOnly = isEditing? "":" readonly='readonly'";
+				
+				//Ternary operator didn't seem to like attributeValue == null ? attributeValue : "";
+				if (null == attributeValue)
+				{
+					attributeValue = ""; 
+				}
+				
+				return String.format("<label>%s: <input value='%s' type='text' class='form-control%s' name='%s' title='%s'%s", 
+						attributeName, attributeValue, formType, attributeName.toLowerCase(), attributeName, readOnly);
+			}
+			%>
 			<div class="col-sm-6 col-sm-offset-3">
 				<form id="userForm" method="post">
 					<div class="form-group">
-						<label>Username: <input <%=editUser != null ? "value='" + editUser.getForename() + "'" : "" %>type="text" class="form-control<%= isEditing?"":" text" %>" name="username" title="Username" <%= isEditing?"":"readonly='readonly'"%>/></label>
-					</div>
-					<div class="form-group">
-						<label>Password: <input type="password" class="form-control" name="password" title="Password"/></label>
-					</div>
-					<div class="form-group">
-						<label>Confirm Password: <input type="password" class="form-control" name="password2" title="Confirm Password"/></label>
+						<%=getUserAttribute(isEditing, "Username", editUser==null?null:editUser.getUsername())%>
 					</div>
+					<%	if(isEditing)
+						{
+							%><div class="form-group">
+								<%=getUserAttribute(isEditing, "Password", "User password here")%>

```

Comment:
value should not be set. maybe change function so that placeholder text can be set.